### PR TITLE
Rename FormHelper methods to be less confusing.

### PR DESCRIPTION
**Pull request by ADmad**
input() => control()
inputs() => controls()
allInputs() => allControls()

### CODE REVIEW

**Comment by ADmad** [comment]
I'll make a PR for the docs also soon.

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -881,12 +881,12 @@ public function label($fieldName, $text = null, array $options = [])
...
      *    be enabled
-     * - `legend` Set to false to disable the legend for the generated input set. Or supply a string
+     * - `legend` Set to false to disable the legend for the generated control set. Or supply a string
      *    to customize the legend text.
-     * @return string Completed form inputs.
+     * @return string Completed form controls.
      * @link http://book.cakephp.org/3.0/en/views/helpers/form.html#generating-entire-forms
      */
-    public function allInputs(array $fields = [], array $options = [])
+    public function allControls(array $fields = [], array $options = [])
```
Comment (jippi) [inline src/View/Helper/FormHelper.php]: why not `allFields` since the argument is called `fields` ? :)

Comment (inoas) [inline src/View/Helper/FormHelper.php]: @jippi See the debate in the issue. 
https://github.com/cakephp/cakephp/issues/9935

Comment (dereuromark) [inline src/View/Helper/FormHelper.php]: Maybe the arguments variable should be called $controls or $controlNames?

Comment (jippi) [inline src/View/Helper/FormHelper.php]: I'm fine wither either, as long as they match :P

Comment (ADmad) [inline src/View/Helper/FormHelper.php]: I was just being consistent with the renames :)  I don't mind changing it to `allFields()`.

Comment (dereuromark) [inline src/View/Helper/FormHelper.php]: allFields() and control() doesnt make any sense.
Either allFields() and field() or allControls() and control() IMO.

Comment (inoas) [inline src/View/Helper/FormHelper.php]: I think it is fine how it is. It does create controls for all fields of the model and is similar to `controls()`. And it doesn't take controls, it takes fieldnames.

Comment (dereuromark) [inline src/View/Helper/FormHelper.php]: That's why $controlNames better :)

Comment (inoas) [inline src/View/Helper/FormHelper.php]: @dereuromark that has been discussed in the issue. https://github.com/cakephp/cakephp/issues/9935#issuecomment-269704851

Fields is saying more than controls. Controls are for selects, radios and are generally what we called "inputs" before as most (but not all) are mapped through the input tag.

**Comment by markstory** [comment]
Looks good to me.

**Pull request merged**